src/pcapi/serialization/utils.py

```diff
@@ -94,6 +94,13 @@ def check_phone_number_format(string: str) -> str:
     return spaceless_string
 
 
+def string_to_boolean(string: str) -> Optional[bool]:
+    try:
+        return {"true": True, "false": False}[string]
+    except KeyError:
+        raise ValidationError("Valeur incorrecte")
```

Comment:
```suggestion
        raise ValidationError("La valeur reçu doit être soit 'true' soit 'false'")
```
Ou chose du genre pour préciser en quoi la valeur est incorrecte